frontend/pages/staking.tsx

```diff
@@ -1064,127 +1062,126 @@ const Staking: NextPage = () => {
                     ))}
                 </Tab.List>
                 <Tab.Panels className="mt-4 sm:mt-11">
-                  {isReady &&
```

Comment:
this was causing hydration errors when wallet was not connected

Reply:
This github diff for this is a disgrace